cfgov/unprocessed/apps/youth-employment-success/js/util.js

```diff
@@ -3,6 +3,21 @@ let UNDEFINED;
 const REDUCER_RETURN_ERROR = 'Reducer must return a state object';
 const INVALID_ARG_ERROR = 'reducers argument must be an object, where each value is a reducer function';
```

Comment:
Should the error sentence be capitalized?

Reply:
I don't know what your policy is for this, `reducers` refers to the name of an argument to a function, and I believe capitalizing it would be in correct

Reply:
👍 alternatively you could say `The "reducers" argument…`. Whatever you think is best.

Reply:
Quotes! so simple, I'll use that 😄